pkg/META

```diff
@@ -6,7 +6,7 @@ package "vfs" (
  directory = "vfs"
  description = "Datakit's VFS description"
  version = "%%VERSION%%"
- requires = ""
+ requires = "fmt astring lwt cstruct logs unix sexplib result"
```

Comment:
I've removed `unix` and `sexplib` and it seems to still work in `utop`. I clearly don't understand what `ocamlobjinfo` does.